tabs/pid_tuning.js

```diff
@@ -741,53 +770,126 @@ TABS.pid_tuning.initialize = function (callback) {
         }
 
         // Getting the DOM elements for curve display
-        var rcCurveElement = $('.rate_curve canvas').get(0);
-        var curveContext = rcCurveElement.getContext("2d");
+        var rcCurveElement              = $('.rate_curve canvas').get(0),
+            rcStickElement              = $('.rate_curve canvas').get(1),
+            curveContext                = rcCurveElement.getContext("2d"),
+            stickContext                = rcStickElement.getContext("2d"),
+            maxAngularVelRollElement    = $('.pid_tuning .maxAngularVelRoll'),
+            maxAngularVelPitchElement   = $('.pid_tuning .maxAngularVelPitch'),
+            maxAngularVelYawElement     = $('.pid_tuning .maxAngularVelYaw'),
+            updateNeeded                = true,
+            maxAngularVel;
+
         rcCurveElement.width = 1000;
         rcCurveElement.height = 1000;
+        rcStickElement.width = 1000;
+        rcStickElement.height = 1000;
+
+        self.updateRatesLabels = function() {
```

Comment:
Instead of adding functions to the object inside other functions, please add them at the top level, like https://github.com/GaryKeeble/betaflight-configurator/blob/3a77c7fadea9c0d73696726cd3820378ee10a819/tabs/pid_tuning.js#L1193